sklearn/metrics/tests/test_pairwise_distances_reduction.py

```diff
@@ -564,6 +564,9 @@ def test_pairwise_distances_reduction_is_usable_for():
     assert not BaseDistancesReductionDispatcher.is_usable_for(
         X_csr, Y_csr, metric="sqeuclidean"
     )
+    assert BaseDistancesReductionDispatcher.is_usable_for(
+        X_csr, Y_csr, metric="euclidean"
+    )
```

Comment:
I am afraid that this will continue failing until both #24542 and #24715 are merged.

Reply:
As discussed IRL, this can be commented for now as it's not blocking this PR.
```suggestion
    # TODO: add this assertion once #24542 and #24715 are merged. 
    # assert BaseDistancesReductionDispatcher.is_usable_for(
    #     X_csr, Y_csr, metric="euclidean"
    # )
```